kit/azure/activity-log/main.tf

```diff
@@ -4,20 +4,56 @@ resource "azurecaf_name" "cafrandom_rg" {
   prefixes      = ["law"]
```

Comment:
note: might not need to use caf module here. RG names don't need to be globally unique, right? 

It's useful for things that must be globally unique in azure like storage account names (see bootstrap module), but i think it just complicates matters for things like naming RGs and LAWs heree

Reply:
Only in the same subscription it have to be unique. I take it out.

Reply:
I changed it to read.